lib/sky-pages-module-generator.js

```diff
@@ -153,7 +139,31 @@ function getSource(skyAppConfig) {
     runtimeModuleExports.push('BBHelpModule');
   }
 
+  const routingSource = [];
+
   if (skyAppConfig.runtime.includeRouteModule) {
```

Comment:
These changes remove any unused types/imports/etc. if the route module is not used.